client/app/scripts/superdesk/directives/sdSlider.js

```diff
@@ -0,0 +1,89 @@
+define(['angular'], function (angular) {
+    'use strict';
+
+    return angular.module('superdesk.slider.directives', []).
+        /**
+         * Slider directive
+         *
+         * Usage:
+         *  <div sd-slider data-value="urgency" data-list="newsvalue" data-unique="value" data-invert="true" data-update="update(item)">
+         *  </div>
+         *
+         * Params:
+         * @scope {Object} value - current selected value, if nothing is selected it will use min value
+         * @scope {Object} list - list of options
+         * @scope {Boolen} disabled - disable or enable slider functionality
+         * @scope {Object} invert - inverts min and max value
+         * @scope {Function} update - callback when slider value is changed
+         *
+         */
+        directive('sdSlider', function () {
+            return {
+                transclude: true,
+                replace: true,
```

Comment:
Is there some alternative for this? Currently this is working, and also I saw that we use this on multiple places.